nanvm-lib/src/js/bitset.rs

```diff
@@ -19,7 +19,9 @@ pub const SIMPLE_SPLIT: (BitSubset64, BitSubset64) = SIMPLE.split(0x0002_0000_00
 
 pub const BOOL: BitSubset64<bool> = SIMPLE_SPLIT.0.cast();
 
+#[allow(dead_code)]
```

Comment:
Could we do this on file/project level?

Reply:
Yes as it turned out we can use #![allow(dead_code)] just once at the top of the file. In this PR we need that in one of mod.rs, but that can also be done at crate level (in lib.rs) - see https://stackoverflow.com/questions/25877285/how-do-you-disable-dead-code-warnings-at-the-crate-level-in-rust.

Reply:
Let's do this on a crate level. We will remove it before publishing the first version.

Reply:
Done - but, I do not know how to turn on a CI requirement on having the build warning-free now on. Let's do that in a separate PR.